src/kinto_http/__init__.py

```diff
@@ -37,5 +37,6 @@ def __init__(self, token, type=None):
         self.type = type or "Bearer"
 
     def __call__(self, r):
+        # Sets auth-scheme  to either Bearer or Basic 
```

Comment:
I think we need to rename this class into `TokenAuth` for better readability.

And we can add:

```py
class BearerTokenAuth(TokenAuth):
    pass
```
for retrocompatibility